src/ui/pages/work/WorkorderScheduling.js

```diff
@@ -23,7 +24,27 @@ const WorkorderScheduling = (props) => {
 
     return (
         <div style={{width: "100%", marginTop: 0}}>
+            <Grid container justify="space-between" spacing={12} style={{margin: "0px"}}>
+                <Grid xs={5}>
```

Comment:
It's missing the 'item' prop. Also, change the 'xs' value to 6, the sum of the items should be 12.
```js
<Grid item xs={6}>
```